### PR TITLE
feat: 🎸 add user registration

### PR DESCRIPTION
**Pull request by Modupef**

### CODE REVIEW

File: app/routes/index.js [hunk 1]
```diff
@@ -1,8 +1,16 @@
+import { Router } from 'express';
+
 import welcomeRoutes from './welcome';
```
Comment (PeculiarE) [inline app/routes/index.js]: What is this route for?

File: app/routes/index.js [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
+
 
-const baseUrl = '/api/v1';
-const routes = app => {
-  app.use(`${baseUrl}`, welcomeRoutes);
-};
+// const baseUrl = '/api/v1';
+// const routes = app => {
+//   app.use(`${baseUrl}`, welcomeRoutes);
+// };
```
Comment (PeculiarE) [inline app/routes/index.js]: Not clear why you commented this out. I'm guessing it's tied to my previous question.

File: app/controllers/auth/index.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      const error = resolveError(e);
+      const dbError = new DBError({
+        status: RESOURCE_CREATE_ERROR_STATUS('USER'),
+        message: e.message
+      });
+      moduleErrLogMessager(dbError);
+      next(error);
+    }
+  }
+}
```
Comment (PeculiarE) [inline app/controllers/auth/index.js:54]: Thought the email confirmation was supposed to happen here?

**Review by PeculiarE — CHANGES_REQUESTED** [review]
Nice job! 

Noticed that ESLint doesn't seem to be warning you about having a newline at the end of your files.

File: app/routes/index.js (hunk 2 above)
Comment (Modupef) [reply app/routes/index.js]: fixed

File: app/controllers/auth/index.js (hunk 3 above)
Comment (Modupef) [reply app/controllers/auth/index.js:54]: yeah. Yet to get confirmation for the team on the email confirmation. I'll fix it in once I do

File: app/routes/index.js (hunk 1 above)
Comment (Modupef) [reply app/routes/index.js]: It is redundant for now

File: app/controllers/auth/index.js (hunk 3 above)
Comment (PeculiarE) [reply app/controllers/auth/index.js:54]: Okies

**Review by PeculiarE — APPROVED** [review]

**Pull request merged**